tasks/testing.rake

```diff
@@ -370,7 +371,7 @@ task :minitest_node_nodejs do
   includes = "-Itest -Ilib -Ivendored-minitest"
 
   sh "ruby -rbundler/setup "\
-     "exe/opal #{includes} #{stubs} -R#{platform} -Dwarning -A --enable-source-location #{filename}"
+     "exe/opal #{includes} #{stubs} -R#{platform} -Dwarning -A --enable-source-location --use-strict #{filename}"
```

Comment:
Compile and test with strict mode enabled

Reply:
If we can control this through an env variable we can add an entry in the test matrix and test both with and without. That said we surely need to start moving in the direction of defaulting to strict mode over time.

Reply:
It's a good idea 👍🏻